TodoList-Capstone/Models/ItemModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ItemModel.swift
+//  TodoList-Capstone
+//
+//  Created by Om Prakash Singh on 08/11/24.
+//
+
+import Foundation
+
+struct ItemModel: Identifiable {
+    let id: UUID
+    let title: String
+    let isCompleted: Bool
+
+    init(id: UUID = UUID(), title: String, isCompleted: Bool) {
```

Comment:
Since you are giving id a default value in the constructor, consider setting it in the initialization on line 11.

Reply:
The init needs to have it as a parameter otherwise the itemModel object which is being returned below wont accept id  as a parameter.